ci/test-pytest.sh

```diff
@@ -2,13 +2,9 @@
 
 cd /vagrant
 
-echo "MINIO_ADDRESS=10.66.60.1:9000
-MINIO_ACCESS_KEY=1234
-MINIO_SECRET_KEY=123456789
-VMCK_API_URL=http://10.66.60.1:10000/v0/
-DEBUG=True
-HOSTNAME=*
-ACS_INTERFACE_ADDRESS=http://10.66.60.1:8000/
-MANAGER_TAG=vagrant-pytest" > .env
+# set the necessary envs if they are not present
+if [ ! -f .env ]; then
+    cp ./examples/.env .
```

Comment:
Q: Why can this happen?

Reply:
If a developer wants to run the testsuite and an `.env` already exists we will overwrite it, and this might break testing if he set the environment differently. Also it is easier to set up the `.env` file for testing in the CI.